### PR TITLE
Added projects

### PR DESCRIPTION
**Pull request by samodovdi**

### CODE REVIEW

File: client/Lykke.Service.BlockchainBalances.Client/AutorestClientMapper.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Service.BlockchainBalances.Client
+{
+    public static class AutorestClientMapper
+    {
+
```
Comment (PaterSantyago) [inline client/Lykke.Service.BlockchainBalances.Client/AutorestClientMapper.cs]: Useless class

File: client/Lykke.Service.BlockchainBalances.Client/BlockchainBalancesServiceClientSettings.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Lykke.Service.BlockchainBalances.Client 
+{
+    public class BlockchainBalancesServiceClientSettings 
+    {
+        public string ServiceUrl {get; set;}
```
Comment (PaterSantyago) [inline client/Lykke.Service.BlockchainBalances.Client/BlockchainBalancesServiceClientSettings.cs:7]: Can we use HttpCheck attribute?

File: src/Lykke.Service.BlockchainBalances.Core/Settings/ServiceSettings/DbSettings.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+namespace Lykke.Service.BlockchainBalances.Core.Settings.ServiceSettings
+{
+    public class DbSettings
+    {
+        public string LogsConnString { get; set; }
```
Comment (PaterSantyago) [inline src/Lykke.Service.BlockchainBalances.Core/Settings/ServiceSettings/DbSettings.cs]: AzureTableCheck attribute?

File: src/Lykke.Service.BlockchainBalances.Services/BlockchainBalancesService.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        public BlockchainBalancesService(
+            NinjaServcieSettings ninjaServiceSettings,
+            EtheriumSamuraiServiceSettings samuraiServiceSettings
+            )
+        {
+            _ninjaUrl = ninjaServiceSettings.ServiceUrl;
+            _samuraiUrl = samuraiServiceSettings.ServiceUrl;
+        }
+
+        public async Task<BalanceModel> GetBalanceAsync(string address, Blockchain blockchain)
```
Comment (PaterSantyago) [inline src/Lykke.Service.BlockchainBalances.Services/BlockchainBalancesService.cs:30]: What if balance for specified parameters is not found?

File: client/Lykke.Service.BlockchainBalances.Client/BlockchainBalancesServiceClientSettings.cs (hunk 2 above)
Comment (samodovdi) [reply client/Lykke.Service.BlockchainBalances.Client/BlockchainBalancesServiceClientSettings.cs:7]: no it's just a class for the settings

File: src/Lykke.Service.BlockchainBalances.Services/BlockchainBalancesService.cs (hunk 4 above)
Comment (samodovdi) [reply src/Lykke.Service.BlockchainBalances.Services/BlockchainBalancesService.cs:30]: Balance property of the BalanceModel will be 0

**Review by PaterSantyago — APPROVED** [review]

**Pull request merged**